tests/python/unittest/test_operator.py

```diff
@@ -7231,6 +7231,18 @@ def f_sm_ce(data, label):
     np_one_hot_label[np.arange(batch_size), np_label] = 1.
     check_symbolic_forward(sym, {'data' : np_data, 'label' : np_label}, [np.array([f_sm_ce(np_sm, np_one_hot_label)])], rtol=1e-3, atol=1e-5)
 
+@with_seed()
+def test_corner_pooling():
+    data = mx.sym.Variable(name='data')
```

Comment:
also call `check_symbolic_forward` and `check_symbolic_backward` - https://mxnet.incubator.apache.org/faq/add_op_in_backend.html#unit-test

Reply:
No problem, thank you!

Reply:
I have added these two functions, ```check_symbolic_forward``` and  ```check_symbolic_backward ```.